app/controllers/weets.js

```diff
@@ -0,0 +1,14 @@
+const geekJokeService = require('../services/geek_joke.service');
```

Comment:
controller is not required according to acceptance criteria

Reply:
Controller removed